styles/index.css

```diff
@@ -0,0 +1,380 @@
+}
+
+.services {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding-top: 80px;
+  z-index: -1;
+  background-color: #fafafa;
+}
+
+.services__image-background {
+  background-color: #0b343d;
+  height: 382px;
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+  z-index: -1;
+}
+
+.services__image {
+  margin-top: 50px;
+  width: 877px;
+  margin-bottom: 55px;
+}
+
+.testimony-card {
+  max-width: 327px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 30px 35px 30px 35px;
+  background-color: white;
+  border: 1px solid #dedede;
+  border-radius: 5px;
+}
+
+.testimony-card__ratings {
+  width: 110px;
+}
+
+.testimony-card__user-name {
+  font-size: 14px;
+  line-height: 24px;
+  font-weight: 600;
+  color: #ffa62b;
+}
+
+.testimony-card__footer {
+  display: flex;
+  gap: 15px;
+}
+
+.testimony-card__user-proffesion {
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.people-section__card-container {
+  display: flex;
+  gap: 31.5px;
+  justify-content: center;
+}
+
+.people-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 80px;
+  padding: 80px 0 80px 0;
+  background-color: #f3f3f3;
+}
+
+.person-card {
+  width: 328px;
+  border: 1px solid #dedede;
+}
+
+.person-card__image {
+  width: 100%;
+}
+
+.person-card__details {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  align-items: center;
+  padding: 30px 0 30px 0;
+}
+
+.person-card__socials-container {
+  display: flex;
+  gap: 15px;
+}
+
+.people-section--white {
+  background-color: white;
+}
+
+.contact-us {
+  background-image: url('../assets/images/contact_us_bg.png');
+  padding: 90px 195px 90px 195px;
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.contact-us__form {
+  display: grid;
+  grid-template-columns: repeat(2, 285px);
+  gap: 20.5px;
+}
+
+.input {
+  padding: 11px 21px 15px 21px;
+  border-radius: 5px;
+  border: 1px solid #bdbdbd;
+  background-color: #f9f9f9;
+}
+
+.input--big {
+  grid-column: span 2;
+  height: 140px;
+}
+
+.consulting {
+  display: flex;
+  padding: 40px 195px 40px 195px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer {
+  padding: 0 195px 0 195px;
+  background-color: #252b42;
+}
+
+.footer__upper-section {
+  padding: 53px 0 53px 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer__lower-section {
+  display: flex;
+  padding: 70px 0 70px 0;
+  justify-content: space-between;
+  color: #fafafa;
+}
+
+.footer__socials-container {
+  display: flex;
+  gap: 20px;
+  padding: 0 40px 0 40px;
+}
+
+.footer__links {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.footer__social-icon {
+  width: 24px;
+}
+
+.footer__input-container {
+  display: flex;
+  padding-top: 20px;
+}
+
+.input--sharp-right-edges {
+  border-radius: 5px 0px 0px 5px;
+}
+
+.footer__input-footer {
```

Comment:
The element contains the block name "footer", it can be confusing if you say footer__input-footer, change this last footer and name it something descriptive, e.g footer__input-item or anything else descriptive. However, you followed BEM instructions.

Reply:
Yeah, sure! 
let me change it to something else